features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2174,6 +2174,27 @@
             <Permissions>/permission/admin/manage/identity/idvp/view</Permissions>
             <Scopes>internal_idvp_view</Scopes>
         </Resource>
+        // TODO: add permission for api resources and permissions
+        <Resource context="(.*)/api/server/v1/api-resources(.*)" secured="true" http-method="GET">
+            <Permissions>/permission/admin/manage/identity/apiresource/view</Permissions>
+            <Scopes>internal_api_resource_view</Scopes>
+        </Resource>
+        <Resource context="(.*)/api/server/v1/api-resources(.*)" secured="true" http-method="POST">
+            <Permissions>/permission/admin/manage/identity/apiresource/create</Permissions>
```

Comment:
These permissions and the permissions defined in [services.xml](https://github.com/wso2/carbon-identity-framework/pull/4854/files#diff-908e93c6d42054b19d8c68a32008ba0034116e3a75e10f6e54460b93e2c07fb2) are different.
Both needs to be same

Reply:
https://github.com/wso2/carbon-identity-framework/pull/4854/commits/8dd16447d6000412e3ff2dd1b2a9ed78249f2d13